stdlib/types.pyi

```diff
@@ -358,6 +358,12 @@ _ReturnT_co = TypeVar("_ReturnT_co", covariant=True)
 
 @final
 class GeneratorType(Generator[_YieldT_co, _SendT_contra, _ReturnT_co]):
```

Comment:
I'd be interested what happens if we just change types.pyi for now, but remove the `typing` base classes to better match runtime.

Reply:
I'm not entirely sure what you're proposing, wouldn't that just crash mypy because typing.Generator wouldn't exist?

Reply:
I meant just using `class GeneratorType:` here, not deleting `typing.Generator`.

Reply:
typing.Generator hasn't been deleted it's just a protocol now